tsl/test/expected/read_only.out

```diff
@@ -306,7 +306,6 @@ SELECT config as comp_job_config
 FROM _timescaledb_config.bgw_job WHERE id = :comp_job_id \gset
 SET default_transaction_read_only TO on;
 CALL _timescaledb_internal.policy_compression(:comp_job_id, :'comp_job_config');
-ERROR:  cannot execute compress_chunk() in a read-only transaction
```

Comment:
Here is one example where the error message is missing but it should be shown.

Reply:
What you ask is contradicting to this enhancement.
We either ignore all kinds of errors or we consider the errors.
Code is such that, it's not possible to ignore specific kinds of errors. Any error during compression policy execution results in an exception which will stop further execution. So in above case if we want to handle exception we cannot proceed further.